pkgs/by-name/ld/ldproxy/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, rustPlatform, fetchFromGitHub, pkg-config, udev, stdenv }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ldproxy";
+  version = "0.31.4";
+
+  # The source of this tools actually comes from the set of tools located in the `esp-rs:embuild` repository
+  # cf: https://github.com/esp-rs/embuild
+  src = fetchFromGitHub {
+    owner = "esp-rs";
+    repo = "embuild";
+    rev = "v${version}";
+    sha256 = "sha256-CPMcFzfP/l1g04sBLWj2pY76F94mNsr1RGom1sfY23I=";
+  };
+
+  # However we are only interested in building the specific crate located at `/ldproxy`
+  # cf: https://github.com/esp-rs/embuild/tree/v0.31.4/ldproxy
+  buildAndTestSubdir = "ldproxy";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    udev
+  ];
```

Comment:
Why is this needed here? `ldproxy` should not need to depend on udev.

Commenting both out, `ldproxy` successfully build without errors, though I still need to test whether it still runs crrectly.

```suggestion
```

Reply:
Thank you I will try to remove those and test.
The original reason for its presence is that I simply reproduced the original PR, I should have tried more to understand it to avoid this issue.

Reply:
Potentially fixed in 0cd4466c2521dbdc3f6d6da63416985877e4c403